mlflow/R/mlflow/.create-test-env.R

```diff
@@ -2,7 +2,7 @@ parent_dir <- dir("../", full.names = TRUE)
 package <- parent_dir[grepl("mlflow_", parent_dir)]
 install.packages(package)
 
-mlflow:::mlflow_maybe_create_conda_env(python_version = "3.6")
+mlflow:::mlflow_maybe_create_conda_env(python_version = "3.7")
```

Comment:
Regardless of this issue, we should use python 3.7 since Mlflow R relies on conda and conda-forge no longer provides packages for python 3.6.

Reply:
I'm still investigating the root cause of this issue.